src/client/src/react/lib.js

```diff
@@ -16,7 +17,12 @@ function setsWonPlayer2(match) {
   return setsWon(match, "player2", "player1");
 }
 
+function isMatchFinished(match) {
```

Comment:
in matchmaker/match -> getMatchWinner(match) 
is this function already  as well 
fyi